src/gui/editors/AutomationEditor.cpp

```diff
@@ -1205,15 +1210,15 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 				/ static_cast<float>( Engine::getSong()->getTimeSigModel().getDenominator() );
 		float zoomFactor = m_zoomXLevels[m_zoomingXModel.value()];
 		//the bars which disappears at the left side by scrolling
-		int leftBars = m_currentPosition * zoomFactor / TimePos::ticksPerBar();
+		int leftBars = barPosition * zoomFactor / TimePos::ticksPerBar();
 
 		//iterates the visible bars and draw the shading on uneven bars
-		for( int x = VALUES_WIDTH, barCount = leftBars; x < width() + m_currentPosition * zoomFactor / timeSignature; x += m_ppb, ++barCount )
+		for( int x = VALUES_WIDTH, barCount = leftBars; x < width() + barPosition * zoomFactor / timeSignature; x += m_ppb, ++barCount )
```

Comment:
May be a long line, too.
```suggestion
		for (int x = VALUES_WIDTH, barCount = leftBars; x < width() + barPosition * zoomFactor / timeSignature; x += m_ppb, ++barCount)
```